src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs

```diff
@@ -117,37 +119,72 @@ public async void IndexGet_GivenRequiredData_ModelMapperShouldBeCalled()
+
+            //assert
+            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetAatfNotesRequest>.That.Matches(g =>
+                g.AatfId.Equals(AatfId) &&
+                g.OrganisationId.Equals(OrganisationId)))).MustHaveHappenedOnceExactly();
+        }
+
+        [Fact]
+        public async void IndexGet_GivenRequiredData_NoteViewModelShouldBeBuilt()
+        {
+            //arrange
+            var notes = Fixture.CreateMany<EvidenceNoteData>().ToList();
+
+            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetAatfNotesRequest>._)).Returns(notes);
+
+            //act
+            await ManageEvidenceController.Index(OrganisationId, AatfId);
+
+            //assert
+            A.CallTo(() => Mapper.Map<EditDraftReturnedNotesViewModel>(
+                A<EditDraftReturnNotesViewModelTransfer>.That.Matches(
+                    e => e.AatfId.Equals(AatfId) && e.OrganisationId.Equals(OrganisationId) &&
+                         e.Notes.SequenceEqual(notes)))).MustHaveHappenedOnceExactly();
+        }
+
         [Fact]
         public async void IndexGet_GivenRequiredData_ModelShouldBeReturned()
         {
             //arrange
-            var organisationId = Guid.NewGuid();
-            var aatfId = Guid.NewGuid();
-            var model = new ManageEvidenceNoteViewModel();
-            A.CallTo(() => Mapper.Map<ManageEvidenceNoteViewModel>(A<ManageEvidenceNoteTransfer>._)).Returns(model);
+            var manageNoteViewModel = new ManageEvidenceNoteViewModel();
+            var evidenceNoteViewModel = new EditDraftReturnedNotesViewModel();
+
+            A.CallTo(() => Mapper.Map<ManageEvidenceNoteViewModel>(A<ManageEvidenceNoteTransfer>._)).Returns(manageNoteViewModel);
+            A.CallTo(() => Mapper.Map<EditDraftReturnedNotesViewModel>(
+                A<EditDraftReturnNotesViewModelTransfer>._)).Returns(evidenceNoteViewModel);
 
             //act
-            var result = await ManageEvidenceController.Index(organisationId, aatfId) as ViewResult;
+            var result = await ManageEvidenceController.Index(OrganisationId, AatfId) as ViewResult;
 
             //assert
-            result.Model.Should().Be(model);
+            var model = result.Model as EditDraftReturnedNotesViewModel;
+
+            model.Should().Be(evidenceNoteViewModel);
+            model.ManageEvidenceNoteViewModel.Should().Be(manageNoteViewModel);
         }
 
         [Fact]
-        public async void IndexGet_GivenAction_DefaultViewShouldBeReturned()
+        public async void IndexGet_GivenAction_EditDraftReturnedNotesOverviewViewShouldBeReturned()
         {
             var organisationId = Guid.NewGuid();
             var aatfId = Guid.NewGuid();
 
             var result = await ManageEvidenceController.Index(organisationId, aatfId) as ViewResult;
 
-            result.ViewName.Should().BeEmpty();
+            result.ViewName.Should().Be("Overview/EditDraftReturnedNotesOverview");
```

Comment:
worth keeping those in a constant and share them between the controller and tests?

Reply:
Sounds good